<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffectsRoutingForm</class>
 <widget class="QWidget" name="EffectsRoutingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    font-family : &quot;Open Sans&quot;;
    font-weight : 550;
    font-size:14pt;
}

#mainFrame{
border: 2px solid white;
    background-color : qlineargradient(spread:reflect, x1:0, y1:0.006, x2:0, y2:1, stop:0 rgba(223, 223, 223, 255), stop:1 rgba(159, 159, 159, 255));
}

QPushButton, #pbSave, #pbCancel, #pbSelect{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(149, 149, 149, 255));
border: 1px solid white;
color: black;
}

QPushButton{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(149, 149, 149, 255));
border: 1px solid white;
color: black;
outline:none;
}

QPushButton:disabled {
	color: grey;
}

QPushButton:focus{ background-color: rgb(239, 159, 31);}

QPushButton:focus:selected{ background-color: rgb(239, 159, 31);}


#lbWidgetTitle{
    border: 1px solid black;
    background-color:rgb(16, 97, 114);
    color: white;
}

#lbCategoryName{
border : 1px solid black;
border-left:none;
background-color: rgb(16, 97, 114);
color:white;
}

#lbCategory{
font-size: 12pt;
border : none;
background-color:rgb(77, 79, 78);
color:white;
}

#lbCategoryNb{
border: 1px solid black;
background-color:rgb(16, 97, 114);
color: white;
}

#frameCategory{
border: none;
color : white;
}

#frameMasterEffect, #frameOutput,#frameCentral{
border:none;
background-color: transparent;
}

#frameMasterEffect1, #frameMasterEffect2, #frameMasterEffect3, #frameMasterEffect4,
#frameEffect1, #frameEffect2, #frameEffect3, #frameEffect4{
border:1px solid white;
background-color: rgb(16, 97, 114);
}

#pbEffect1, #pbEffect2, #pbEffect3, #pbEffect4{
border: 1px solid white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(149, 149, 149, 255));
color: back;
}

#pbEditEffect1, #pbEditEffect2, #pbEditEffect3, #pbEditEffect4,
#pbMasterEditEffect1, #pbMasterEditEffect2, #pbMasterEditEffect3{
	color: white;
	border: none;
	background-color:transparent;
}

#frameEditEffect1, #frameEditEffect2, #frameEditEffect3, #frameEditEffect4,
#frameMasterEditEffect1, #frameMasterEditEffect2, #frameMasterEditEffect3{
background-color:rgb(16, 97, 114);
border:1px solid white;
border-left: none;
} 

#frameEffectProp1, #frameEffectProp2, #frameEffectProp3, #frameEffectProp4,
#frameMasterEffectProp1, #frameMasterEffectProp2, #frameMasterEffectProp3{
background-color:rgb(16, 97, 114);
border:1px solid white;
border-right : none;
color: white;
}


QCheckBox{
    background-color    : transparent;
	border:1px solid black;
    outline:none;
}

QCheckBox::indicator { 
    width               : 49px; 
    height              : 19px; 
}

QCheckBox::indicator:unchecked {
    background-color : rgb(202, 36, 36);
}

QCheckBox::indicator:checked {
    background-color : rgb(84, 189, 58);
}

#lbEffectName1, #lbEffectName2, #lbEffectName3, #lbEffectName4,
#lbMasterEffectName1, #lbMasterEffectName2, #lbMasterEffectName3, #lbMasterEffectName4{
border : none;
color: white;
}

#frameEffects{
	border:none;
}

#lbSampleNb1, #lbSampleNb2, #lbSampleNb3, #lbSampleNb4{
background: transparent;
padding-left:50px;
border:none;
color:black;
}
#frameSample1, #frameSample2, #frameSample3, #frameSample4{
border:1px solid white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(149, 149, 149, 255));
color: back;
}

#lbSampleName1, #lbSampleName2, #lbSampleName3, #lbSampleName4{
background: transparent;
border:none;
color:black;
}

#lbLinkedEffect1, #lbLinkedEffect2, #lbLinkedEffect3, #lbLinkedEffect4{
background-color : white;
border: 1px solid black;
color:black;
}

#lbSection1, #lbSection2, #lbSection3{
border: 1px solid black;
background-color:rgb(77, 79, 78);
color: white;
font-size : 12pt;
}

#lbEffect1, #lbEffect2, #lbEffect3, #lbEffect4{
border: 1px solid white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(149, 149, 149, 255));
color: back;
font-size:12pt;
}

QGraphicsView{
background-color: transparent;
border: none;
}

QScrollArea, #scrollAreaWidgetContents {
background-color: transparent; 
border: none;
}

QScrollBar:horizontal {
border :   1px solid black;
background :   black;
height : 10px;
margin : 0 0 0 0px;
}

QScrollBar::handle:horizontal {    
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(149, 149, 149, 255));
min-width              :   20px;
margin: 0px 0px 0px 0px;
}

QScrollBar::add-line:horizontal,QScrollBar::sub-line:horizontal  {
border                  :   none;
background              :   transparent;
height                  :   0px;
margin                  :   0 0 0 10px;
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
border                  :   none;
width                   :   0px;
height                  :   0px;
background              :   white;
margin                  :   0 0 0 10px;
}

#pbEffectLink1,#pbEffectLink2,#pbEffectLink3,#pbEffectLink4,
#pbMasterEffectLink1,#pbMasterEffectLink2,#pbMasterEffectLink3{
background-color: rgb(239, 159, 31);
margin-left: 1px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="mainFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>480</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>14</pointsize>
       <weight>68</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="frameTop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>58</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>58</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>14</pointsize>
          <weight>68</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>8</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lbWidgetTitle">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Open Sans</family>
               <pointsize>14</pointsize>
               <weight>68</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string> EFFECTS ROUTING</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="font">
              <font>
               <family>Open Sans</family>
               <pointsize>14</pointsize>
               <weight>68</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="frameCategory">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_28">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lbCategoryNb">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Open Sans</family>
                <pointsize>14</pointsize>
                <weight>68</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>001</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbCategoryName">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Piano Layers &amp; Strings</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="verticalSpacer_9">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="pbFrame">
        <property name="minimumSize">
         <size>
          <width>778</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>778</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbInsertEFX">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>126</width>
             <height>58</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>126</width>
             <height>58</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>INSERT EFX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbMasterEFX">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>126</width>
             <height>58</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>126</width>
             <height>58</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>MASTER EFX</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbCancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>126</width>
             <height>58</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>126</width>
             <height>58</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>CANCEL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSave">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>126</width>
             <height>58</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>126</width>
             <height>58</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <weight>68</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>SAVE</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1470</width>
           <height>312</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_25">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QGraphicsView" name="graphicsView">
            <property name="minimumSize">
             <size>
              <width>1470</width>
              <height>312</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1470</width>
              <height>312</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
